src/Apps/Feature/FeatureApp.tsx

```diff
@@ -48,6 +41,10 @@ const FeatureApp: React.FC<FeatureAppProps> = ({ feature }) => {
             feature.sets.edges.map(
               ({ node: set }) => set && <FeatureSet key={set.id} set={set} />
             )}
+
+          <Separator my={3} />
+
+          <Footer />
```

Comment:
Its unfortunate that this has to be added in this way right now, but we have a bit of conflict in the A/B test, so it can't go [in the new AppShell](https://github.com/artsy/reaction/blob/master/src/Apps/Components/AppShell.tsx#L60) by default just yet.